index.d.ts

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+
+type DateInfo = {
+    day: number;
+    month: number;
+    year: number;
+}
+
+type Range = {
+    from: DateInfo | null;
+    To: DateInfo | null;
+}
+
+export interface DatePickerProps {
+    value: Range | DateInfo[] | null;
+    onChange: (newValue: Range | DateInfo[] | null) => null;

```

Comment:
The return type of `onChange` function should be `void`.

Reply:
I'm agree with void but docs of this component told me this:

![image](https://user-images.githubusercontent.com/35793921/68698277-d761d380-0595-11ea-973d-f0227678a93c.png)

We can fix both if u want

Reply:
`void` is a better option as it can accept both null and undefined.